benchmarks/src/main/scala/scalaz/effect/BubbleSortBenchmarks.scala

```diff
@@ -0,0 +1,60 @@
+package scalaz.effect
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import scala.concurrent.duration.Duration
+import scala.collection.immutable.Range
+
+import IOBenchmarks.unsafePerformIO
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+class BubbleSortBenchmarks {
+  @Param(Array("1000"))
+  var size: Int = _
+
+  def createTestArray: Array[Int] = Range.inclusive(1, size).toArray.reverse
```

Comment:
Why aren't all of these importing `Array`, `Int`, etc? We turned off no implicit prelude in the main scalaz.

Reply:
I didn't import that change, to make it easier to use this lib in a non-scalaz environment. We are using std everywhere as opposed to scalaz core where using our own data types is preferred.

Reply:
I think we should make compiler options as consistent as possible. Obviously we can't enforce Scalazzi here and we can't depend on any other project, but strict compiler options consistent with Scalaz will help ensure uniform coding standards across the Scalaz projects.

Reply:
Ok, I'll do it in a separate PR, it's not a priority to get snapshots published right now IMHO.

Reply:
Agreed.